proxy/api/src/http.rs

```diff
@@ -40,16 +40,28 @@ pub fn api(
     enable_fixture_creation: bool,
 ) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
     let avatar_filter = path("avatars").and(avatar::get_filter());
-    let control_filter = path("control").and(control::filters(
-        ctx.clone(),
-        selfdestruct,
-        enable_fixture_creation,
-    ));
+    let control_filter = path("control")
+        .and(with_unsealed_guard(ctx.clone()))
+        .untuple_one()
```

Comment:
Can't we move the `untuple_one` into the body of `with_unsealed_guard`? It would spare us the reptition and help avoid confusion when used.